apps/api/docs/production-setup.md

```diff
@@ -82,7 +82,7 @@ Make sure to replace the above example values with appropriate values as per you
   module.exports = {
     apps: [
       {
-        name: 'osmo-notify', // Name of your application
+        name: 'osmoX', // Name of your application
```

Comment:
Update to `OsmoX`

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b